rapid7_attackerkb/plugin.spec.yaml

```diff
@@ -0,0 +1,361 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_attackerkb
+title: Rapid7 AttackerKB
+description: AttackerKB is a knowledge base of vulnerabilities and informed opinions on what makes them valuable (or not) targets for exploitation
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://attackerkb.com/landing
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- rapid7
+- attackerkb
+- vulnerability
+- cve
+hub_tags:
+  use_cases: [remediation_management, vulnerability_management]
+  keywords: [rapid7, attackerkb, vulnerability, cve]
+  features: []
+connection:
+  credentials:
+    title: API Key
+    description: API key from account e.g. YYDHZKByMaDTMmY4ZC12MmUxLTkyTTBtY2UxUzkxNjbbYWI2OMzLYjATHjABZ9x3MUhyVUEzMWF1N0E5QThDOEhsQTRrMW1GVDZWUGVaDnA9
+    type: credential_secret_key
+    required: true
+    example: YYDHZKByMaDTMmY4ZC12MmUxLTkyTTBtY2UxUzkxNjbbYWI2OMzLYjATHjABZ9x3MUhyVUEzMWF1N0E5QThDOEhsQTRrMW1GVDZWUGVaDnA9
+  max_pages:
+    title: Max Pages
+    description: Max pages returned, default 100
+    type: integer
+    required: false
+    default: 100
+    example: 10
+types:
```

Comment:
It's recommended that the type section for a plugin spec comes before the connection